cmd/expiration-mailer/main.go

```diff
@@ -205,15 +205,21 @@ func (m *mailer) processCerts(allCerts []core.Certificate) {
 			parsedCerts = append(parsedCerts, parsedCert)
 		}
 
+		if len(parsedCerts) == 0 {
```

Comment:
A test for this behavior, i.e. no `Error sending nag emails` log messages, still seems to be missing.